core/remotes/docker/config/hosts.go

```diff
@@ -54,7 +54,10 @@ type hostConfig struct {
 
 	header http.Header
 
-	// TODO: Add credential configuration (domain alias, username)
+	// TODO: Add credential configuration (token, helper, ...)
+	credentialDomain string
+	username         string
+	password         string
```

Comment:
Secrets are intentionally not included here

Reply:
Ah got it, I will remove the password field from the config. 

BTW what kind of credentials can we specify in the host config? only identity token and credential helper?